examples/server/ktor-server/src/main/kotlin/com/expediagroup/graphql/examples/server/ktor/GraphQLModule.kt

```diff
@@ -16,6 +16,7 @@
 
 package com.expediagroup.graphql.examples.server.ktor
 
+import com.expediagroup.graphql.generator.extensions.*
```

Comment:
can we use specific imports instead of wildcard one?

Reply:
Ah yes, my IDE was configured according to ktor's weird recommendation to use star imports everywhere
https://ktor.io/docs/code-style.html#with-star-imports

fixed